modules/Hoot/tools/clipDataset.js

```diff
@@ -128,7 +127,11 @@ export default class ClipDataset {
                     } else if ( d.name === 'outputName' ) {
                         that.createLayerNameField( d3.select( this ), layer );
                     } else {
+                        var folderId = Hoot.layers.findBy( 'id', Number(layer.id) ).folderId;
+                        d.combobox = [ d.combobox.find( function( l ) { return l.id === folderId; } ) ]
+                            .concat( d.combobox.filter( function( l ) { return l.id !== folderId; } ).sort() );
                         that.createFolderListCombo( d3.select( this ), d );
+                        d3.select( this ).property( 'value', Hoot.folders.findBy( 'id', folderId ).name );
```

Comment:
What happens if you also set the property '_value' to equal the result of folderId above? I feel like this may make is so that you dont need all the calls to the find() function below

Reply:
:checkered_flag: